rest/proxy/error.go

```diff
@@ -0,0 +1,32 @@
+package proxy
+
+import (
+	"context"
+	"github.com/fabric8-services/fabric8-wit/errors"
+	"net/http"
+)
+
+// ConvertHTTPErrorCode converts an http status code to an instance of type error
+func ConvertHTTPErrorCode(ctx context.Context, statusCode int, responseBody string) error {
```

Comment:
Ctx param is not used. So, it can be removed from the method.

Reply:
yeah, let me remove this. Initially thought of using this after seeing this https://github.com/fabric8-services/fabric8-wit/blob/master/errors/errors.go#L30 . Should refactor that as a separate PR

Reply:
https://github.com/fabric8-services/fabric8-wit/pull/2026/commits/4572164393700adb5e8914cb99bbfd65cf5744d7